modules/troubleshooting/pages/log-files.adoc

```diff
@@ -1,107 +1,111 @@
-I@20210709 13:56:52.219  (SessionManager.java:197) Abort and clear all sessions...
-I@20210709 13:56:52.220  (SessionManager.java:204) All sessions aborted.
-I@20210709 13:56:52.224  (GsqlHAHandler.java:283) switched to new leader m1
+$ gadmin log
+ADMIN  : /home/tigergraph/tigergraph/log/admin/ADMIN#1.out
+ADMIN  : /home/tigergraph/tigergraph/log/admin/ADMIN.INFO
+CTRL   : /home/tigergraph/tigergraph/log/controller/CTRL#1.log
+CTRL   : /home/tigergraph/tigergraph/log/controller/CTRL#1.out
+...
+TS3    : /home/tigergraph/tigergraph/log/ts3/TS3_1.log
+TS3    : /home/tigergraph/tigergraph/log/ts3/TS3_1.out
+TS3SERV: /home/tigergraph/tigergraph/log/ts3serv/TS3SERV#1.out
+ZK     : /home/tigergraph/tigergraph/log/zk/ZK#1.out
+ZK     : /home/tigergraph/tigergraph/log/zk/zookeeper.log
 ----
 
-== Monitor request history
+Use the command `gadmin log <service name>` to just get the logs for a specific service:
 
-All requests made to TigerGraph's REST endpoints are recorded by the RESTPP logs and Nginx logs. Information available in the logs includes:
+[source, console]
+----
+$ gadmin log gpe
+GPE    : /home/tigergraph/tigergraph/log/gpe/GPE_1#1.out
+GPE    : /home/tigergraph/tigergraph/log/gpe/log.INFO
+----
 
-* Timestamp of the request
-* API request parameters
-* Request Status
-* User information (when RESTPP authentication is turned on)
+The `log.INFO` file contains messages logged by the application code.
+The `.out` log contains the redirection of the process output, and is used for debugging significantly less frequently than `log.INFO`.
 
-RESTPP is responsible for many tasks in the TigerGraph internal architecture and records many internal API calls, which can be hard to distinguish from manual requests. When xref:user-access:enabling-user-authentication.adoc#_enable_restpp_authentication[RESTPP authentication is on], the RESTPP log will record the user information and mark a call if it is made by an internal API. Therefore, you can use the command below to filter for manual requests:
+[CAUTION]
+The log format differs between the `.out` and `INFO` logs.
+It also differs between certain TigerGraph services.
+An internal project to unify log formats is ongoing.
 
+Log formats also vary across the different components.
+In folders where logs are checked often, such as `restpp`, `gsql`, and `admin`,  there are symbolic links that help you quickly get to the most recent log file of that category:
+
+* `log.INFO`
+** Contains regular output and errors
+* `log.ERROR`
+** Contains errors only
+* `<component_name>.out`
+** Contains all output from the component process. Current `.out` logs have the form `<service name>.out`.
+Historical logs have the form `<service name>-old-YYYY-MM-DDTHH-MM-SS.fff.out`
+
+* `log.WARNING` or `log.DEBUG`
+** `log.WARNING` contains warnings and all error level messages
+** In the``gsql`` folder, `log.DEBUG` contains very specific information used only for certain error situations
```

Comment:
GSQL no longer logs at the debug level by default. 
May be worth mentioning that customer can configure debug level logging using SERVICE.BasicConfig.LogConfig.LogLevel config value